### ISSUE TITLE
Prefer backslash over two spaces for line breaks

### ISSUE THREAD

**Issue by thernstig** [DavidAnson/markdownlint#455]
Most editors and code review tools often highlight empty spaces as a "fault". Either marking them red or similar.

Since commonmark supports both two spaces or a backslash to render a line-break, could this tool add a rule to only recommend backslash, due to the above mentioned reasons?

```markdown
# Two spaces
test  
test

# Backslash
test\
test
```

**Comment by DavidAnson** [comment]
Yes, probably like: line-break-style and spaces/backslash/consistent

**Comment by thernstig** [comment]
And I think backslash should be default, or consistent. Due to backslash helping with the trailing white-space issue. But I assume that would be a breaking change.

**Comment by DavidAnson** [comment]
Right, in rules where "consistent" is an option, it is the default.
